apps/app/src/views/routes/Settings/views/TreatmentTab.tsx

```diff
@@ -24,7 +24,7 @@ import { ADD_TO_CATALOG } from '../../../../mutations';
 import { SplitLayout } from '../../../components/SplitLayout';
```

Comment:
this file is responsible for rendering treatments in the catalog, not all treatments.